docs: init a vuepress documentation (#2075)
ovh/manager: docs/guide/architecture.md

# Architecture

## General information

> The monorepo is powered by two technologies:

Reviewer: i would just say "The monorepo is powered by :", not sure if technologies is the right word :)

- [Yarn](https://yarnpkg.com) — Fast, reliable, and secure dependency management.
- [Lerna](https://lerna.js.org/) — A tool for managing JavaScript projects with multiple packages.

Several [workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) are configured and you can find the list in the root [`package.json`](https://github.com/ovh/manager/blob/master/package.json#L7-L12) file.

```json
{
  "workspaces": [
    "packages/components/*",
    "packages/manager/apps/*",
    "packages/manager/modules/*",
    "packages/manager/tools/*"
  ],
  …,
}
```

## Languages and Framework

The **Manager** application is using these following languages:

- HTML
- CSS ([Less.js](http://lesscss.org/), [SCSS](https://sass-lang.com/))
- JavaScript

> [AngularJS](https://angularjs.org/) is widely used as the MVW Framework to build [Single Page Applications](https://en.wikipedia.org/wiki/Single-page_application).

Reviewer 2: ```suggestion
[AngularJS](https://angularjs.org/) used to be a widelys used MVW Framework to build [Single Page Applications](https://en.wikipedia.org/wiki/Single-page_application).
```

::: tip Information

> We are exploring some new JavaScript Frameworks to speed up the development workflow.

Reviewer 2: ```suggestion
We are exploring some new solutions to speed up the development workflow.
```
